### PR TITLE
fix: updates to gridview tabs styling issue 246

### PR DESCRIPTION
**Pull request by djmcquillan**
- fixes: #246 
- fixes: #249 

https://user-images.githubusercontent.com/10619673/212128837-cc77cc82-5197-4a9c-afdd-579fff6dc17a.mov

### CODE REVIEW

**Comment by juliaxbow** [comment]
Awesome, thanks! Minor nitpick - can we swap the order ("Detailed View" first)? Eventually want the site to default to detailed view (discussed in Notion [here](https://www.notion.so/pl-strflt/2022-12-08-StarMaps-Touchbase-cdf651185f164504a6cc1246dfa67e82#da23998dabc44fb5bf81c36bec7fd231))

**Comment by djmcquillan** [comment]
@juliaxbow 
with current code reordering the tabs will also default to Detailed View.  Let me know if we want to wait on defaulting to detail view and I will update

File: components/roadmap-grid/Roadmap.module.css [hunk 1]
```diff
@@ -48,7 +48,7 @@
   z-index: 4;
 }
 
-.nested > .item:not(.group) {
+.nested>.item:not(.group) {
```
Comment (SgtPooki) [inline components/roadmap-grid/Roadmap.module.css:51]: remove

```suggestion
.nested > .item:not(.group) {
```

File: components/roadmap-grid/Roadmap.module.css [hunk 2]
```diff
@@ -117,13 +117,6 @@
   padding: 10px;
 }
 
-.timelineBox {
-  padding: 0.25rem;
-}
```
Comment (SgtPooki) [inline components/roadmap-grid/Roadmap.module.css:122]: this conflicts with your other PR changes

File: pages/roadmap/[...slug].tsx [hunk 3]
```diff
@@ -222,10 +222,10 @@ export default function RoadmapPage(props: InferGetServerSidePropsType<typeof ge
   return (
     <>
       <PageHeader />
-      <div style={{ overflowY: 'auto', height: 'calc(100vh - 100px)' }}>
+      <div style={{ overflowY: 'auto', height: 'calc(100vh - 120px)', paddingTop: '28px' }}>
         {issueDataState.ornull != null && <StarmapsBreadcrumb currentTitle={issueDataState.ornull.title.value} />}
         <ErrorNotificationDisplay errorsState={starMapsErrorsState} issueDataState={issueDataState}/>
-        <Box pt={5} pr={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} pl={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} >
+        <Box pr={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} pl={{ base:"30px", sm:"30px", md:"60px", lg:"120px" }} >
```
Comment (SgtPooki) [inline pages/roadmap/[...slug].tsx:228]: nit: why is this needed? I'm concerned that the height between this and the `.scrollable` component need to be maintained across separate files, and I would like to see us get to a point where we centralize these calculations and handle this better.

**Review by SgtPooki — CHANGES_REQUESTED** [review]

**Review by SgtPooki — APPROVED** [review]

File: pages/roadmap/[...slug].tsx (hunk 3 above)
Comment (djmcquillan) [reply pages/roadmap/[...slug].tsx:228]: this is a valid concern, but note the change did not introduce the height calc. it only updated it to adjust for the margin/padding changes

**Pull request merged**